projects/i18n/languages/russian/kit.ts

```diff
@@ -53,4 +53,6 @@ export const TUI_RUSSIAN_LANGUAGE_KIT: LanguageKit = {
         drop: 'Поместите файл сюда',
         dropMultiple: 'Поместите файлы сюда',
     },
+    unavailablePdf:
+        'В настройках вашего браузера установлена загрузка файлов PDF вместо их отображения в окне',
```

Comment:
Let's ask our designer to come up with better description ? 

I'm not sure this is a clear explanation.

Reply:
fixed